sale_commission/models/res_partner.py

```diff
@@ -36,6 +36,13 @@ class ResPartner(models.Model):
     settlements = fields.One2many(
         comodel_name="sale.commission.settlement", inverse_name="agent",
         readonly=True)
+    commission_ids = fields.One2many(comodel_name='res.partner.agent',
+                                     inverse_name='partner_id',
+                                     string='Agents')
+    default_commission = fields.Boolean(
```

Comment:
This field should be in res.partner.agent level, not here. The idea is:

* You set the agents at customer (partner) level
* You check for each agent if they used the default commission set as agent level
* Or you specify a specific one for this customer